### PR TITLE
app-misc/radeontop: bump to 1.3

### PR DESCRIPTION
**Pull request by mercuriete**
Closes: https://bugs.gentoo.org/765529
Signed-off-by: Abel Garcia Dorta <mercuriete@yahoo.es>

### CODE REVIEW

**Comment by mercuriete** [comment]
I think that CI error is a false positive because I didn't touch that package.

Can somebody have a look on that? I don't have the knowledge to understand what is happening.

**Comment by thesamesam** [comment]
@mercuriete Please rebase on master, this was an unrelated issue. Thanks!

**Comment by mercuriete** [comment]
@thesamesam thanks I already did it 1 minute ago.

File: app-misc/radeontop/radeontop-1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (mm1ke) [inline app-misc/radeontop/radeontop-1.3.ebuild:1]: You need to update the header (or use repoman commit)

**Review by mm1ke — COMMENTED** [review]
With the updated header i'm fine with the bump :)

**Comment by mercuriete** [comment]
I will do It when I have time.
Probably tonight. Thanks for the feedback

File: app-misc/radeontop/radeontop-1.3.ebuild (hunk 1 above)
Comment (mercuriete) [reply app-misc/radeontop/radeontop-1.3.ebuild]: @mm1ke 
I did a change on the header by hand...
I am not sure if the change is ok.
Thank you for your review.

**Comment by mercuriete** [comment]
@mm1ke gentle ping.

**Comment by mm1ke** [comment]
Thanks, i'm fine now with the bump.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**